test/e2e/clusterlogging_test.go

```diff
@@ -51,7 +52,7 @@ func TestClusterLogging(t *testing.T) {
 
 		for _, collector := range []string{"fluentd", "rsyslog"} {
 			t.Run(collector, func(t *testing.T) {
-				t.Parallel()
+				//				t.Parallel()
```

Comment:
@ewolinetz can you speak to this? It parallelizes the tests?

Reply:
i believe so, though it did seem that they ran in series during a `make test-e2e`

Reply:
I observed that when use `t.Parallel()`, the value of `collector` is always equal to `rsyslog` in `clusterLoggingInitialDeploymentTest` and `clusterLoggingUpgradeTest`.  Found a gist about this: https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721